laalaa/apps/advisers/views.py

```diff
@@ -152,6 +152,20 @@ class UploadSpreadsheetForm(forms.Form):
     xlfile = forms.FileField(label="Spreadsheet")
 
 
+def import_progressive_advisors(xls_file, user):
```

Comment:
So... I think the task is not to import progressive advisers, but rather to progressively import advisers... 😅

So I think we'd be better suited with `progressively_import_advisers` or even just `import_advisers`.
Also note the spelling of "adviser" here

Reply:
amended to import_advisers

Reply:
amended method to `import_advisers`